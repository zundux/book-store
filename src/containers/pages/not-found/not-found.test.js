import React from "react";
import NotFound from "./not-found";

describe("Not found page", () => {

  it("not found page", () => {
    const component = shallow(<NotFound/>);
    matchSnapshot(component);
    // todo check page header
    // todo check page title
  });
});