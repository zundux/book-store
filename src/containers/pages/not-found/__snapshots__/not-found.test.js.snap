// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page not found page 1`] = `
<div>
  <h2>
    Page not found
  </h2>
</div>
`;
